data-catalog-policies-app/src/main/resources/db/migration/V1__initial_version.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE LEGAL_BASIS (
+    legal_basis_id SERIAL PRIMARY KEY,
+    description VARCHAR(500) NOT NULL
+);
+
+CREATE TABLE PURPOSE (
+    purpose_id SERIAL PRIMARY KEY,
+    description VARCHAR(500) NOT NULL
+);
+
+CREATE TABLE POLICY (
+    policy_id SERIAL PRIMARY KEY,
+    information_type_id INTEGER NOT NULL,
+    purpose_id INTEGER NOT NULL,
+    legal_basis_id INTEGER NOT NULL,
+    CONSTRAINT FK_POLICY_PURPOSE FOREIGN KEY (purpose_id)
```

Comment:
Here we can use field name too but we don't have circular references in our model so it's fine.